app/components/Dashboard.tsx

```diff
@@ -108,8 +115,22 @@ const Dashboard: React.FC<Props> = ({ query: queryKey }) => {
         </section>
         {isAdmin && (
           <section>
-            <h3 id="admin-heading">Administration</h3>
-            <nav aria-labelledby="admin-heading"></nav>
+            <div id="admin-heading">
+              <h3>Administration</h3>
+            </div>
+            <nav aria-labelledby="admin-heading">
+              <Link passHref href={getMetabaseRoute()}>
+                <BCGovLink target="_blank">Data Insights (Metabase)</BCGovLink>
+              </Link>
+              <Link
+                passHref
+                href={decodeURI(
+                  getSupportEmailMailTo("CIF App: Report a problem!").pathname
+                )}
```

Comment:
Do you know why the original path is URL-encoded at that point?

Reply:
since we are passing `mailto` subject like `?subject=...` the question mark turns into `%3F` in the JSX and that causes `mailto` to not properly populated with the subject.

Reply:
I can remove it with no effect on my end, and it's not required in the `Footer` component, I don't think `decodeURI(...)` is doing anything here

Reply:
Hmmm strange, I've tried that on my end and the subject was getting added to the email rather than in the subject field. I will try again and post the screenshot if I face it again.

Reply:
You are right, it doesn't need it. I don't know why it was acting up at that time 😄